app/src/components/ChoiceCard.tsx

```diff
@@ -149,13 +156,16 @@ function ChoiceCard({
     <div
       role="region"
       className="h-100 border-0 bg-transparent"
-      onClick={onCardClick}
```

Comment:
moving onClick to the actual card element (on the `region`, you can click the background on accident and trigger an onClick event)